sdk/storage/Azure.Storage.DataMovement.Files.Shares/CHANGELOG.md

```diff
@@ -3,8 +3,12 @@
 ## 12.2.0-beta.1 (Unreleased)
 
 ### Features Added
+- Added support for preserving NFS properties and permissions in ShareFiles and ShareDirectories for Share-to-Share copy transfers.
```

Comment:
NIT - Add spaces between ShareFiles and ShareDirectories to be "Share Files" and "Share Directories" because this is a changelog

Reply:
Same below